global_body_planner/include/global_body_planner/planning_utils.h

```diff
@@ -231,11 +316,96 @@ void getInterpPlan(const FullState &start_state,
                    const PlannerConfig &planner_config);
 
 // Terrain-based heuristics
-double getZFromState(const State &s, const PlannerConfig &planner_config);
 double getPitchFromState(const State &s, const PlannerConfig &planner_config);
 double getDzFromState(const State &s, const PlannerConfig &planner_config);
 void setDz(State &s, const PlannerConfig &planner_config);
 void setDz(State &s, const Eigen::Vector3d &surf_norm);
+inline bool isInMap(const Eigen::Vector3d &pos,
+                    const PlannerConfig &planner_config) {
+  // Uncomment to use grid_map
+  // return planner_config.terrain_grid_map.isInside(pos.head<2>());
+  return planner_config.terrain.isInRange(pos[0], pos[1]);
+}
```

Comment:
Not sure this is autoformatted, but one line space between functions would be nice

Reply:
Done.